### DISCUSSION TITLE
Update Now Playing via POST/GET

### DISCUSSION THREAD

**Discussion by BeyondChrispy** [AzuraCast/AzuraCast#6209] [Q&A] [answered]
I use FLAC to send my live stream to Azuracast as a DJ/Streamer.  FLAC doesn't currently support metadata - is there a way to update the song artist/title metadata via POST/GET or similar to get my song data onto the mount points?

**Comment by BusterNeece** [comment, marked as answer]
This is possible if you send an authenticated `POST` request to `your-azuracast-url/api/station/(station_id)/nowplaying/update` with a JSON body and/or form parameters for `title` and `artist`.

**Comment by BeyondChrispy** [comment]
Thanks, sorted!
